packages/graphql/tests/integration/directives/fulltext/fulltext-query.int.test.ts

```diff
@@ -2039,7 +2039,7 @@ describe("@fulltext directive", () => {
             try {
                 const result = await session.run(indexQueryCypher);
 
-                expect(result.records[0].get("result")).toEqual({
+                expect(result.records[0]?.get("result")).toEqual({
```

Comment:
This file is a good example of where we could use the generic param available to us in `session.run`